rero_ils/config.py

```diff
@@ -303,16 +303,19 @@ def _(x):
         indexer_class=IlsRecordIndexer,
         record_serializers={
             'application/json': (
+                'invenio_records_rest.serializers' ':json_v1_response'
+            ),
+            'application/rero+json': (
```

Comment:
rero+json can be replaced by can-delete+json

Reply:
done, there is possibly something to adapt on the admin views for this as well (?)